lib/pong.js

```diff
@@ -32,6 +33,28 @@ var pong = {
     });
   },
 
+  findPlayers: function(user_names, cb) {
+    var findPlayer = function(user_name) {
+      var deferred = Q.defer();
+      pong.findPlayer(user_name, function (err, player) {
+        if (err) {
+          deferred.reject(err);
+        } else {
+          deferred.resolve(player);
+        }
+      });
+      return deferred.promise;
+    };
```

Comment:
I'm not familiar with Mongoose but apparently you can return promises from queries: https://github.com/Automattic/mongoose/blob/4.0.1/lib/promise.js

It might be nicer if the `findPlayer` method directly returned a promise instead of having to wrap it like this.